.github/workflows/integration_tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     runs-on: ubuntu-latest
     environment: backend-validation
     container:
-      image: ghcr.io/nvidia/cuda-quantum:latest
+      image: nvcr.io/nvidia/nightly/cuda-quantum:latest
```

Comment:
Do any authentication steps need to be added to make this work?

Reply:
No, finally it is possible to download the image without signing in.